<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NFL Pamphlet</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Desktop/football1.png</normaloff>../../../../Desktop/football1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>880</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dataPage">
     <widget class="NFLDataTable" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1600</width>
        <height>811</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>155</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Team Name</string>
       </property>
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stadium Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Seating Capacity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>City</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>State</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Conference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Division</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Surface Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stadium Roof Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date Opened</string>
       </property>
      </column>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>1160</x>
        <y>819</y>
        <width>431</width>
        <height>38</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>221</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Total Stadium Capacities:</string>
       </property>
      </widget>
      <widget class="QLabel" name="totalCapacityLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>0</y>
         <width>191</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="contactPage">
     <widget class="QStackedWidget" name="contactPageStack">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1600</width>
        <height>880</height>
       </rect>
      </property>
      <widget class="QWidget" name="regularContactPage">
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>120</y>
          <width>710</width>
          <height>620</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>710</width>
           <height>620</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;Please contact us through the methods below&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;Phone: (123) 111 - 222 - 3333&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;Email: NeedHelp@Saddleback.edu&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;Help Hours:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;Monday - Friday&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;8:00am - 4:00pm&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic; color:#000000;&quot;&gt;subject to holiday changes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="adminContactPage">
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>120</y>
          <width>710</width>
          <height>620</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>710</width>
           <height>620</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;ADMIN CONTACT PAGE&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;Please contact us through the methods below&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;Phone: (321) 999 - 888 - 7777&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;Email: AdminHelp@Saddleback.edu&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="helpPage">
     <widget class="QLabel" name="helpLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1600</width>
        <height>861</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Home:&lt;/span&gt; To reach the Home page, click &amp;quot;Menu&amp;quot; in the top left corner, then click &amp;quot;Home&amp;quot;.&lt;/p&gt;&lt;p&gt;After clicking this option, you should be automatically dirrected to the &amp;quot;Home&amp;quot; page. &lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Contact Us:&lt;/span&gt; To reach the Contact Us page, click &amp;quot;Menu&amp;quot; in the top left corner, then click &amp;quot;Contact Us&amp;quot;.&lt;/p&gt;&lt;p&gt;After clicking this option, you should be automatically dirrected to the &amp;quot;Contact Us&amp;quot; page.&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Display Conference:&lt;/span&gt; To refine the &amp;quot;Home&amp;quot; page to show one of these options;&lt;/p&gt;&lt;p&gt;----&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Display all:&lt;/span&gt; Click &amp;quot;Menu&amp;quot; in the top left corner, then hover over &amp;quot;Display Conference&amp;quot;.&lt;/p&gt;&lt;p&gt;Please click on the option &amp;quot;All&amp;quot;.&lt;/p&gt;&lt;p&gt;All conferances should now be displayed on the &amp;quot;Home&amp;quot; page.&lt;/p&gt;&lt;p&gt;----&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Display American Football Conferance:&lt;/span&gt; Click &amp;quot;Menu&amp;quot; in the top left corner, then hover over &amp;quot;Display Conference&amp;quot;.&lt;/p&gt;&lt;p&gt;Please click on the option &amp;quot;American Football Conferance&amp;quot;.&lt;/p&gt;&lt;p&gt;American Football Conferances should now be displayed on the &amp;quot;Home&amp;quot; page.&lt;/p&gt;&lt;p&gt;----&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Display National Football Conferance:&lt;/span&gt; Click &amp;quot;Menu&amp;quot; in the top left corner, then hover over &amp;quot;Display Conference&amp;quot;.&lt;/p&gt;&lt;p&gt;Please click on the option &amp;quot;National Football Conferance&amp;quot;.&lt;/p&gt;&lt;p&gt;National Football Conferances should now be displayed on the &amp;quot;Home&amp;quot; page.&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Log in:&lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;*Option only of importance to Administrators*&lt;/span&gt;&lt;/p&gt;&lt;p&gt;If you are an Administrator hoping to find and use the login page, click &amp;quot;Menu&amp;quot; in the top left corner, then click &amp;quot;Login&amp;quot;.&lt;/p&gt;&lt;p&gt;After clicking &amp;quot;Login&amp;quot; a window will pop up asking you to input a password.&lt;/p&gt;&lt;p&gt;Entering an incorrect password will result in a pop up window.&lt;/p&gt;&lt;p&gt;If login is successful there should be no pop ups and a drop down titled &amp;quot;Admin&amp;quot; will appear next to menu in the top left corner.&lt;/p&gt;&lt;p&gt;If you are an Administrator and do not have the password, please speak to other Administrators about your dilemna.&lt;/p&gt;&lt;p&gt;Refer to this drop down for all Admin tools.&lt;/p&gt;&lt;p&gt;Contact Administrators for more assistance, thank you.&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;*If you need further assisstance, please refer to the &amp;quot;Contact Us&amp;quot; page (located by clicking &amp;quot;Menu&amp;quot;)*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QWidget" name="adminHelpWidget" native="true">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>80</y>
        <width>601</width>
        <height>461</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>581</width>
         <height>431</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <widget class="QLabel" name="adminHelpLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>561</width>
          <height>411</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Admin Help&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Load New Entries:&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; Allows the upload of a .CSV file of choice.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;-Example of data within .CSV file: San Diego Sailors,Qualcomm Stadium,....&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Show Original List:&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; Shows the data of the original .CSV file.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Show Updated List:&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; Shows the original data plus any additionally &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;uploaded data together.&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic;&quot;&gt;-Example of use:&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; Click &amp;quot;Admin&amp;quot; then click &amp;quot;Load New Entries&amp;quot;, &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;select your .CSV file of choice.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Click &amp;quot;Show Updated List&amp;quot; to view the new data that has now &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;been added to the Original List.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;To view original data list please select &amp;quot;Show Original List&amp;quot;.&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuDisplay_Conference">
     <property name="title">
      <string>Display Conference</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Desktop/football1.png</normaloff>../../../../Desktop/football1.png</iconset>
     </property>
     <addaction name="actionNo_Conferences_Found"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionHome"/>
    <addaction name="actionlogin"/>
    <addaction name="actionContact_Us"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="menuDisplay_Conference"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionLoad_New_Entries"/>
    <addaction name="actionShow_Original_List"/>
    <addaction name="actionShow_Updated_List"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAdmin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlogin">
   <property name="icon">
    <iconset>
     <normaloff>../../../../Desktop/login.png</normaloff>../../../../Desktop/login.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionContact_Us">
   <property name="icon">
    <iconset>
     <normaloff>../../../../Desktop/contactUs.png</normaloff>../../../../Desktop/contactUs.png</iconset>
   </property>
   <property name="text">
    <string>Contact Us</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset>
     <normaloff>../../../../Desktop/Home1.png</normaloff>../../../../Desktop/Home1.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionLoad_New_Entries">
   <property name="text">
    <string>Load New Entries</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>../../../../Desktop/Help.png</normaloff>../../../../Desktop/Help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionShow_Original_List">
   <property name="text">
    <string>Show Original List</string>
   </property>
  </action>
  <action name="actionShow_Updated_List">
   <property name="text">
    <string>Show Updated List</string>
   </property>
  </action>
  <action name="actionNo_Conferences_Found">
   <property name="text">
    <string>No Conferences Found</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NFLDataTable</class>
   <extends>QTableWidget</extends>
   <header>nfldatatable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
